podcasts/ShowNotesUpdater.swift

```diff
@@ -4,10 +4,14 @@ import PocketCastsUtils
 
 class ShowNotesUpdater {
     class func updateShowNotesInBackground(podcastUuid: String, episodeUuid: String) {
-        if FeatureFlag.newShowNotesEndpoint.enabled {
+        if FeatureFlag.newShowNotesEndpoint.enabled || FeatureFlag.transcripts.enabled {
```

Comment:
I believe we don't need the `|| FeatureFlag.transcripts.enabled` do we?

Regardless of the transcripts, we want to continue caching chapters/show notes, so it can be the same as it always has.

I feel like this might lead to a regression when removing `newShowNotesEndpoint`. If we leave `transcripts` there cache for downloaded episodes will stop working.

Reply:
Ok, will remove this then.

Reply:
Better yet,sShould I separate the transcripts loading completely  from the `newShowNotesEndpoint`, do we need to link transcripts  to the new endpoint? 
I see the endpoint is the one that provides the data for transcripts but was the flag just done because of the way we update notes or to control the use of the endpoint?

Reply:
Not really. But I plan to remove this flag anyway, so I think we can leave this for another task. Wdyt?

Reply:
Ok will, just updated the code [here](https://github.com/Automattic/pocket-casts-ios/pull/1852/commits/1b351697bf0bf1c1a88ec60d6626a7afddec0a61) and the PR description to reflect the changes.